src/main/java/org/reactome/release/qa/check/CoVEntityDiseaseCheck.java

```diff
@@ -0,0 +1,98 @@
+import org.gk.model.ReactomeJavaConstants;
+import org.reactome.release.qa.annotations.SliceQACheck;
+import org.reactome.release.qa.common.AbstractQACheck;
+import org.reactome.release.qa.common.QACheckerHelper;
+import org.reactome.release.qa.common.QAReport;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * This QA check was mostly used during the CoV-1-to-CoV-2 inference process. For now it is being kept, but may
+ * be removed, given it was for a specific curations (August 2020).
+ *
+ * This checks for instances that have either a CoV-1 or CoV-2 species without the corresponding disease attribute.
+ * For CoV-1 instances, they would be missing 'severe acute respiratory disease' (9678120). For CoV-2 instances, they
+ * would be missing 'COVID-19' (9683912).
+ *
+ * @author jcook
+ */
+
+@SliceQACheck
+public class CoVEntityDiseaseCheck extends AbstractQACheck {
+
+    @Override
+    public QAReport executeQACheck() throws Exception {
+        QAReport report = new QAReport();
+
+        // Fetches all Events and PhysicalEntities in DB
+        Collection<GKInstance> eventsAndPhysicalEntities = new ArrayList<>();
+        eventsAndPhysicalEntities.addAll(dba.fetchInstancesByClass(ReactomeJavaConstants.Event));
+        eventsAndPhysicalEntities.addAll(dba.fetchInstancesByClass(ReactomeJavaConstants.PhysicalEntity));
+
+        for (GKInstance instance : eventsAndPhysicalEntities) {
+            // Get all species, relatedSpecies, and disease instances within instance being checked
+            Set<Long> speciesDbIds = QACheckerHelper.getSpeciesAndRelatedSpeciesDbIds(instance);
+
+            Set<Long> diseaseDbIds = new HashSet<>();
+            if (instance.getSchemClass().isValidAttribute(ReactomeJavaConstants.disease)) {
+                for (GKInstance disease : (Collection<GKInstance>) instance.getAttributeValuesList(ReactomeJavaConstants.disease)) {
+                    diseaseDbIds.add(disease.getDBID());
+                }
+            }
+
+            // Check for missing disease attributes for CoV instances.
+            if (hasCoVSpeciesWithoutDisease(speciesDbIds, diseaseDbIds)) {
+                report.addLine(getReportLine(instance, speciesDbIds));
+            }
+        }
+
+        report.setColumnHeaders(getColumnHeaders());
+        return report;
+    }
+
+    private boolean hasCoVSpeciesWithoutDisease(Set<Long> speciesDbIds, Set<Long> diseaseDbIds) {
+        return hasCoV1SpeciesWithoutDisease(speciesDbIds, diseaseDbIds) || hasCoV2SpeciesWithoutDisease(speciesDbIds, diseaseDbIds);
+
+    }
+
+    private boolean hasCoV1SpeciesWithoutDisease(Set<Long> speciesDbIds, Set<Long> diseaseDbIds) {
+        return speciesDbIds.contains(QACheckerHelper.COV_1_SPECIES_DB_ID) && !diseaseDbIds.contains(QACheckerHelper.COV_1_DISEASE_DB_ID);
+    }
+
+    private boolean hasCoV2SpeciesWithoutDisease(Set<Long> speciesDbIds, Set<Long> diseaseDbIds) {
```

Comment:
Similar to https://github.com/reactome/release-qa/pull/26/files#r545253485 could this be named to be `...WithoutCoV2Disease`?